tests/test_es_base_block.py

```diff
@@ -1,23 +1,87 @@
 import logging
-from ..es_base_block import ESBase
+from unittest.mock import patch, MagicMock
 
 from nio.util.support.block_test_case import NIOBlockTestCase
-from ..tests import elasticsearch_running
+from nio.common.signal.base import Signal
+
+from ..es_base_block import ESBase
 
 
 class TestESBase(NIOBlockTestCase):
+
     """ Tests basic elasticsearch functionality provided by
     ESBase class
     """
 
-    def setUp(self):
-        self._outcome.success = elasticsearch_running()
-        if self._outcome.success:
-            super().setUp()
+    @patch('elasticsearch.Elasticsearch.ping')
+    def test_connected(self, ping_method):
+        blk = ESBase()
+        self.configure_block(blk, {
+            "log_level": "DEBUG"
+        })
+        blk.connected()
+        ping_method.assertCalledOnceWith()
+
+    def test_query_execute_and_return(self):
+        """ Test that if queries return signals they get notified """
+        blk = ESBase()
+        self.configure_block(blk, {})
+        blk.start()
+
+        # Execute query will return a list of 2 signals
+        blk.execute_query = MagicMock(return_value=[Signal(), Signal()])
```

Comment:
For Dynamo I moved this type of mocking to the class level so all tests had it. That might be appropriate here.